src/main/java/fi/aalto/cs/apluscourses/ui/courseproject/CourseProjectView.form

```diff
@@ -86,7 +86,7 @@
           <grid row="3" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
         </constraints>
         <properties>
-          <text value="Select the language used for the course:"/>
+          <text value="Select the language you use for the course assignments. (Finnish is recommended if you know Finnish.)"/>
```

Comment:
can you please place this into properties bundle

Reply:
Yes will do :+1: